weblate_web/templates/download.html

```diff
@@ -46,23 +46,23 @@ <h2 class="section-title left">{% trans "Cloud or virtualization images" %}</h2>
                 </div>
                 <div class="col">
                     <div class="d-logo"><img src="{% static "img/openshift.svg" %}" alt="#" /></div>
-                    <div class="title">{% trans "Running Weblate on OpenShift platform" %}</div>
-                    <a href="https://docs.weblate.org/en/latest/admin/deployments.html#openshift" class="button">{% trans "Openshift guide" %}</a>
+                    <div class="title">{% trans "Running Weblate on the OpenShift platform" %}</div>
+                    <a href="https://docs.weblate.org/en/latest/admin/deployments.html#openshift" class="button">{% trans "OpenShift guide" %}</a>
                 </div>
                 <div class="col">
                     <div class="d-logo"><img src="{% static "img/weblate-stack.png" %}" alt="#" /></div>
-                    <div class="title">{% trans "BitNami Weblate stack" %}</div>
+                    <div class="title">{% trans "Bitnami Weblate stack" %}</div>
                     <p>{% trans "Native installation for most platforms and support for public cloud providers." %}</p>
                     <ul>
-                        <li><a href="https://bitnami.com/stack/weblate/cloud/google">Google Cloud Platform</a></li>
+                        <li><a href="https://bitnami.com/stack/weblate/cloud/google">Google Cloud platform</a></li>
```

Comment:
It's AFAIK called "Google Cloud Platform", see https://cloud.google.com/products/

Reply:
I thought I had changed that back.